app/src/main/java/com/fstyle/androidtrainning/data/service/config/MoviesApi.java

```diff
@@ -0,0 +1,37 @@
+package com.fstyle.androidtrainning.data.service.config;
+
+import com.fstyle.androidtrainning.data.model.Movie;
+import com.fstyle.androidtrainning.data.service.response.GetListNowPlayingResponse;
+import com.fstyle.androidtrainning.data.service.response.GetListPopularResponse;
+import com.fstyle.androidtrainning.data.service.response.GetListTopRatedResponse;
+import com.fstyle.androidtrainning.data.service.response.GetListUpcomingResponse;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Path;
+import retrofit2.http.Query;
+
+/**
+ * Created by Admin on 12/08/17.
+ */
+
+public interface MoviesApi {
+    @GET("/3/movie/upcoming")
+    Call<GetListUpcomingResponse> getListUpcomingMovies(@Query("api_key") String apiKey,
```

Comment:
WARNING: [AvoidDuplicateLiterals] The String literal "api_key" appears 5 times in this file; the first occurrence is on line 19
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals